builder/frameworks/espidf.py

```diff
@@ -634,6 +644,79 @@ def build_espidf_bootloader():
         LINKFLAGS=["-T", "ulp_main.ld"]
     )
 
+if is_arduino_enabled(sdk_params):
+    ARDUINO_FRAMEWORK_DIR = platform.get_package_dir("framework-arduinoespressif32")
+    assert ARDUINO_FRAMEWORK_DIR and isdir(ARDUINO_FRAMEWORK_DIR)
+    print 'Including Arduino-ESP32 from %s' % ARDUINO_FRAMEWORK_DIR
+    env.Prepend(
```

Comment:
Can we use default build script https://github.com/espressif/arduino-esp32/blob/master/tools/platformio-build.py ?

You can call it with https://github.com/platformio/platform-espressif32/blob/develop/builder/frameworks/arduino.py#L33

Reply:
Unfortunately replacing the block with
```
SConscript(join(ARDUINO_FRAMEWORK_DIR, "tools", "platformio-build.py"))
```
results in:
```
*** Multiple ways to build the same target were specified for: C:\trains\platform-espressif32\examples\espidf-arduino-wifi\.pioenvs\esp32dev\partitions.bin  (from ['C:\\users\\mdunston\\.platformio\\packages\\framework-espidf\\components\\partition_table\\partitions_singleapp.csv'] and from ['partitions\\default.csv'])
File "C:\users\mdunston\.platformio\packages\framework-arduinoespressif32\tools\platformio-build.py", line 222, in <module>
```

Reply:
@valeros please review